diff --git a/src/UpdateCheck.lua b/src/UpdateCheck.lua
index 7d3f8bec..9f439c09 100644
--- a/src/UpdateCheck.lua
+++ b/src/UpdateCheck.lua
@@ -4,6 +4,9 @@
 -- Module: Update Check
 -- Checks for updates
 --
+return "none"
+
+--[[
 local connectionProtocol, proxyURL, noSSL = ...

 local xml = require("xml")
@@ -338,3 +341,4 @@ opFile:close()

 ConPrintf("Update is ready.")
 return updateMode
+]] --
